static/js/markdown.js

```diff
@@ -5,6 +5,8 @@ import _ from "lodash";
 import * as fenced_code from "../shared/js/fenced_code";
 import marked from "../third/marked/lib/marked";
 
+import * as blueslip from "./blueslip";
```

Comment:
I believe we're trying to share this file, so we don't want to import `blueslip` or other modules not from `shared/js` here. 

i think the convention is to use `throw new Error` to throw an exception if needed.